docs/spec_draft.md

```diff
@@ -1791,6 +1792,41 @@ and produces a `result` tensor.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.power
+
+### Semantics
+
+Performs element-wise exponentiation of `lhs` tensor by `rhs` tensor and
+produces a `result` tensor.
```

Comment:
1. Can you refer to the IEEE-754 semantics?
2. What is the semantics of a complex power?
3. What happens to integer overflow?

Reply:
thanks